src/API.jl

```diff
@@ -1582,4 +1582,18 @@ function upgrade_manifest(ctx::Context = Context())
     return nothing
 end
 
+"""
+    auto_gc()
+
+Query whether Pkg will automatically [`gc`](@ref) packages and artifacts.
+"""
+auto_gc() = _auto_gc_enabled[]
```

Comment:
Can we use the same API as `GC.enable` (where it returns the previous state) to avoid having two methods?

Reply:
Sure, done.